_posts/2017/2017-03-10-creating-a-sales-analytics-app-with-ruby-on-rails-2.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "Creating a Sales Analytics App with Ruby on Rails, part 2"
+date: "2017-03-10 07:00:00"
+image: blog-header/rails-train-path-straight.jpg
+categories: tech-stories
+authors: ["Unai M."]
+---
+
+A couple of days ago we started writing about how and why we started our Sales analytics app. Today we want to talk about some issues we encountered in our way and how we solved them.
+
+## Issues on the development process
+
+As I wrote earlier, D3 is a really powerful engine to do all sorts of graphs but is by no means simple.
+This and the fact that we (backend, frontend and design) had separated tasks for the graphs created issues whenever we needed to change anything.
```

Comment:
I don't understand this sentence "tasks for the graphs created issues..." > what does that relate to?